Source/Core/Core/IOS/IOS.h

```diff
@@ -110,7 +110,7 @@ void WriteReturnValue(s32 value, u32 address);
 class Kernel
 {
 public:
-  Kernel();
+  Kernel(IOSC::ConsoleType console_type = IOSC::ConsoleType::Retail);
```

Comment:
Should probably be marked `explicit` to avoid unintended implicit conversions.

Reply:
As far as I understand, not marking this explicit will allow implicit conversions from IOSC::ConsoleType to Kernel, which would be confusing.

Reply:
You won't get implicit conversions from integers and such, but you'll still get implicit conversions from `IOSC::ConsoleType`.